settings.gradle.kts

```diff
@@ -203,3 +203,9 @@ include(":system-tests:runtimes:azure-data-factory-transfer-consumer")
 include(":system-tests:runtimes:azure-storage-transfer-consumer")
 include(":system-tests:azure-tests")
 include(":system-tests:azure-data-factory-tests")
+
+include(":tooling:module-domain")
```

Comment:
Why is `tooling` not part of the `core` module as this also contains basic functionality as `boot` and `base`?

Reply:
Tooling is separate from the core runtime, e.g. it is not required for the latter's operation. On a related note, @ndr-brt is working on a proposal that separates the core further.